### ISSUE TITLE
Don't install metawrap via metawrap-mg

### ISSUE THREAD

**Issue by yqy6611** [ursky/metaWRAP#548]
This is a suggestion. 

metawrap-mg is totally outdated. Moreover, all of dependencies can actually work with python 3 without revising module scripts of metawrap. 

Prepare a cleaner conda environment with python 3 and manually install those dependencies except for prokka and salmon can solve majority problems.  For prokka and salmon, prepare separate environments for each of them and redirect the locations in scripts.

For some small auxiliary python 2 scripts we can prepare another python 2 environment and redirect python interpreter in those scripts. Of course it will be better if you can revise those scripts using python 3.

**Comment by bachelot34** [comment]
When you say do not use metawrap-mg, are you saying we should follow this step described in the installation document: 

# Note: ordering is important
conda config --add channels defaults
conda config --add channels conda-forge
conda config --add channels bioconda
conda config --add channels ursky

conda install biopython blas=2.5 blast=2.6.0 bmtagger bowtie2 bwa checkm-genome fastqc kraken=1.1 krona=2.7 matplotlib maxbin2 megahit metabat2 pandas quast r-ggplot2 r-recommended samtools=1.9 seaborn spades trim-galore

and then add prokka and salmon afterwards?

Thank you

**Comment by yqy6611** [comment]
> When you say do not use metawrap-mg, are you saying we should follow this step described in the installation document: 
> 
> 
> 
> # Note: ordering is important
> 
> conda config --add channels defaults
> 
> conda config --add channels conda-forge
> 
> conda config --add channels bioconda
> 
> conda config --add channels ursky
> 
> 
> 
> conda install biopython blas=2.5 blast=2.6.0 bmtagger bowtie2 bwa checkm-genome fastqc kraken=1.1 krona=2.7 matplotlib maxbin2 megahit metabat2 pandas quast r-ggplot2 r-recommended samtools=1.9 seaborn spades trim-galore
> 
> 
> 
> and then add prokka and salmon afterwards?
> 
> 
> 
> Thank you
> 
> 

Yes and no need to add ursky channel. And don't add spades 4.0.0 in this environment due to compatibility issues. If you really hope to use it, install it in a separate environment like prokka and salmon.

**Comment by bachelot34** [comment]
Thank you! When you say I will have to adapt the scripts after creating new environments for salmon and prokka, which scripts are you referring to? Some of the metawrap scripts, or my own scripts?
